.env.example

```
@@ -82,25 +95,30 @@ ENGAGEMENT_WEIGHT_TECHNICAL=0.15 # Scales with how many technical keywords appea
+ENGAGEMENT_REACTION_EMOJI=👍 # Emoji reaction used when ignoreMode=react.
 ENGAGEMENT_MIN_THRESHOLD=0.5 # Engage only if the final score is >= this threshold.
 ENGAGEMENT_PROBABILISTIC_LOW=0.4 # Lower bound (inclusive) of the optional LLM refinement band.
 ENGAGEMENT_PROBABILISTIC_HIGH=0.6 # Upper bound (inclusive) of the optional LLM refinement band.
 ENGAGEMENT_ENABLE_LLM_REFINEMENT=false # Enables refinement pass (currently a no-op stub).
+DISCORD_BOT_LOG_FULL_CONTEXT=false # Enable only when debugging large-context behavior
 
 # Optional web server configuration
 # Canonical local backend origin for bot API calls and the web Vite dev proxy target
+DATA_DIR=/data # Persistent backend data directory
+HOST=:: # Host/interface the backend server binds to
+PORT=3000 # Port the backend server listens on
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)